src/main/kotlin/exercise2/task1/FindPairOfHighestSum.kt

```diff
@@ -18,7 +18,17 @@ import org.jetbrains.exercise2.task3.findPairWithBiggestDifference
  */
 
 internal fun List<Int>.findHighestSumPair(): Pair<Int, Int> {
-    TODO("Implement me!!")
+    var par = Pair(this.min(), this.min())
+    var sum = par.first + par.second
+    for (x in this) {
+        for (y in this) {
+            if (y != x && (x + y) > sum) {
```

Comment:
nit: `x != y` može da se izbegne tako što prva petlja iterira od prvog do predposlednjeg elementa, a druga petlja iterira od `x+1` do poslednjeg elementa liste.